yjit.rb

```diff
@@ -63,7 +63,7 @@ def self.graphviz_for(iseq)
       highlight = ->(comment) { "<b>#{comment}</b>" }
       linebreak = "<br align=\"left\"/>\n"
 
-      buff = ''
+      buff = ''.dup
```

Comment:
```suggestion
      buff = +''
```

Since there is no pragma in the beginning of the file, the strings are only frozen if the command line option is passed, and the `+` method would only dup if they are frozen.